src/main/java/homework/seven/ArrayValueCalculator.java

```diff
@@ -0,0 +1,30 @@
+package homework.seven;
+
+public class ArrayValueCalculator {
+    String[][] array = new String[4][4];
+
+    public int doCalc(String[][] array) throws ArraySizeException, ArrayDataException {
+        int suma = 0;
+        this.array = array;
```

Comment:
Why do you need a variable `array` and this line 8? We may have static method doCalc

Reply:
i deleted this.array = array;